packages/google_sign_in/google_sign_in_ios/ios/Classes/FLTGoogleSignInPlugin.m

```diff
@@ -78,52 +79,46 @@ - (instancetype)initWithSignIn:(GIDSignIn *)signIn {
 
 - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
   if ([call.method isEqualToString:@"init"]) {
-    NSString *path = [[NSBundle mainBundle] pathForResource:@"GoogleService-Info" ofType:@"plist"];
-    if (path) {
-      NSMutableDictionary<NSString *, NSString *> *plist =
```

Comment:
plist parsing logic moved into `configurationClientIdArgument`.  These properties are now stored in a `GIDConfiguration` and do not hang off the sign in object.